pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -34,15 +34,17 @@ python3.pkgs.buildPythonApplication rec {
   doCheck = false;
 
   postPatch = ''
+    # The generated udev rules point to /bin/chmod, which does not exist in NixOS
+    substituteInPlace src/hhd/controller/lib/hide.py \
+      --replace-fail /bin/chmod ${toybox}/bin/chmod
+
     # handheld-daemon contains a fork of the python module `hid`, so this hook
     # is borrowed from the `hid` derivation.
+    substituteInPlace src/hhd/controller/lib/hid.py \
+      --replace-fail libhidapi ${hidapi}/lib/libhidapi
+
     hidapi=${hidapi}/lib/
```

Comment:
Why isn't the `substituteInPlace` using this variable?

Reply:
Not sure since I did this a while ago, but iirc I just copied the substituteInPlace right on top and then changed it. Easy change tho so I'll do that rn

Reply:
Except that it's duplicating the path now.  If that path ever changed, you could change one and not the other - it would break and you might not realize it.

In some senses, this is just cargo-culted over from `hid`; however, as long as the variable is there, it should be used in both places.  Moreover, since this does mirror the `hid` build rule, it should keep symmetry with the `hid` derivation (which does the test before calling `subInPlace`).